.clang-format

```diff
@@ -0,0 +1,117 @@
+---
+Language:        Cpp
+# BasedOnStyle:  Google
```

Comment:
Please use BasedOnStyle: Google - then specify the overriding additionals rather than specifying the entire default style. As new versions of clang-format come out we don't want to have to update this.